src/Refactoring-Core/RBPullUpMethodRefactoring.class.st

```diff
@@ -282,8 +340,7 @@ RBPullUpMethodRefactoring >> privateTransform [
 	self
 		copyDownMethods;
 		pullUpMethods;
-		removePulledUpMethods;
-		removeDuplicateMethods
```

Comment:
Why was `removeDuplicateMethods` removed? Is it because it was moved to driver?

Reply:
Exactly, removeDuplicateMethods is now offered as a choice (as presented in the video above)